db/migrate/20210510130434_create_participant_events.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateParticipantEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :participant_events, id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+      t.string :item_type, null: false
+      t.string :event, null: false
+      t.string :whodunnit
+      t.json :object
+      t.json :object_changes
```

Comment:
What is `object` exactly?

Reply:
> What is `object` exactly?

Standard for paper_trail (as also used in the "versions" table that it was based on.
https://github.com/paper-trail-gem/paper_trail

Reply:
Could we visually separate the parts of migration that are 'ours' and the parts that are for papertrail? Maybe with a comment for the latter saying "Fields for paper_trail"?